packages/ag-charts-website/src/utils/pages.ts

```diff
@@ -90,6 +90,11 @@ export const getContentRootFileUrl = ({ isDev }: { isDev?: boolean } = {}): URL
     return new URL(contentRoot, import.meta.url);
 };
 
+export const getExampleRootFileUrl = ({ isDev = getIsDev() }: { isDev?: boolean } = {}): URL => {
+    const root = isDev ? '../../../..' : '../../../../..';
```

Comment:
There's a method above for this

```suggestion
    const root = getRootUrl().pathname;
```